tests/unit/test_base.py

```diff
@@ -139,7 +139,10 @@ def test_link():
             note=fmf_id['note'])]
 
     # Invalid links and relations
-    with pytest.raises(SpecificationError, match='Invalid link'):
+    with pytest.raises(
+            SpecificationError,
+            match="Field 'link' can be string, fmf id or list of their commbinations,"
```

Comment:
Typo: double 'm'

Reply:
Fixed in 9529a776134c84fe8c7428edbfe79f2eb70cfaa4